app/dashboard/templates/profiles/profile.html

```diff
@@ -47,7 +47,7 @@
       {% endif %}
     </div>
 
-    <div class="container-fluid bg-light">
+    <div class="container-fluid">
```

Comment:
removing bg-light will turn the bg white right?

Reply:
Yes